foodsaving/stores/tests/test_feedback_api.py

```diff
@@ -0,0 +1,133 @@
+        cls.feedback = {
+            'given_by': cls.member.id,
+            'about': cls.past_pickup.id,
+            'weight': 2,
+            'comment': 'asfjk'
+        }
+
+    def test_create_feedback_fails_as_non_user(self):
+        """
+        Non-User is not allowed to give feedback.
+
+        Test:
+        1. user gives feedback to pickup
+        2. make sure that response is NOT valid
+        """
+        response = self.client.post(self.url, self.feedback, format='json')
+
+        self.assertEqual(
+            response.status_code, status.HTTP_403_FORBIDDEN, response.data)
+
+    def test_create_feedback_fails_as_non_group_member(self):
+        """
+        User is not allowed to give feedback when not a member of the stores group.
+
+        Test:
+        1. log user in
+        2. user gives feedback to pickup
+        3. make sure that response is NOT valid
+        """
+        self.client.force_login(user=self.user)
+        response = self.client.post(self.url, self.feedback, format='json')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.data)
+        self.assertEqual(response.data, {'about': ['You are not member of the store\'s group.']})
+
+    def test_create_feedback_fails_as_non_collector(self):
+        """
+        Group Member is not allowed to give feedback when he is not assiged to the
+        Pickup.
+
+        Test:
+        1. log user in as group member
+        2. user gives feedback to pickup
+        3. feedback NOT created
+        """
+        self.client.force_login(user=self.member)
+        response = self.client.post(self.url, self.feedback, format='json')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.data)
+        self.assertEqual(response.data, {'about': ['You aren\'t assign to the pickup.']})
+
+    def test_create_feedback_works_as_collector(self):
+        """
+        Member is allowed to give feedback when he is assiged to the Pickup.
+
+        Test:
+        1. log user in as group member
+        2. user gives feedback to pickup
+        3. feedback created
+        """
+        self.client.force_login(user=self.collector)
+        response = self.client.post(self.url, self.feedback, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
+
+    def test_list_feedback_fails_as_non_user(self):
+        """
+        Non-User is NOT allowed to see list of feedback
+        """
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN, response.data)
+
+    def test_list_feedback_fails_as_non_group_member(self):
+        """
+        Non-Member is NOT allowed to see list of feedback
+        """
+        self.client.force_login(user=self.user)
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND, response.data)
```

Comment:
I would expect an empty list instead of `404`, similar to [here](https://github.com/yunity/foodsaving-backend/blob/3280d295173176a57ea0a5049bc1ad8686143ed7/foodsaving/stores/tests/test_pickupdates_api.py#L437)

Explanation: the request goes to `/api/feedback/`, which is not group-specific. It will return the list of all feedback the user has access to, which is an empty list in this case.

Reply:
We have done that yesterday evening. You can look it up in our follow-up commit that I pushed.